### ISSUE TITLE
g_param_spec_static_name_canonical" False positive when using I_

### ISSUE THREAD

**Issue by A6GibKm** [bilelmoussaoui/gobject-lint#20]
e.g
```
gtk/gtk/gtkstacksidebar.c:388:28
  ...
     /**
     * GtkStackSidebar:stack:
     *
     * The stack.
     */
    obj_properties[PROP_STACK] =
  >      g_param_spec_object (I_("stack"), NULL, NULL,
                             GTK_TYPE_STACK,
                             G_PARAM_READWRITE|G_PARAM_STATIC_STRINGS|G_PARAM_EXPLICIT_NOTIFY);
  ...
  error: Property name 'I_("stack")' is not canonical (contains underscores). With G_PARAM_STATIC_NAME this will cause: g_param_spec_internal: assertion '!(flags & G_PARAM_STATIC_NAME) || is_canonical (name)' failed. Use 'I-("stack")' instead
  rule: g_param_spec_static_name_canonical
```
